SPARQLWrapper/Wrapper.py

```diff
@@ -1162,7 +1172,9 @@ def _validate_format(format_name, allowed, mime, requested):
 
         # TODO. In order to compare properly, the requested QueryType (SPARQL Query Form) is needed. For instance,
         # the unexpected N3 requested for a SELECT would return XML
-        if "content-type" in self.info():
+        if "content-type" not in self.info():
```

Comment:
This looks like it can go into a separate PR even though it is just changing order of conditions.

Reply:
934e334